plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -5801,13 +6290,92 @@ img.help_tip {
 				font-family: "WooCommerce";
 				content: "\e600";
 			}
+		}
+	}
+}
+
+.woocommerce {
+	ul.wc-tabs {
+		width: 20%;
+		float: left;
+		line-height: 1em;
+		padding: 0 0 10px;
+		position: relative;
+		background-color: #fafafa;
+		border-right: 1px solid #eee;
+		box-sizing: border-box;
+
+		&::after {
+			content: "";
+			display: block;
+			width: 100%;
+			height: 9999em;
+			position: absolute;
+			bottom: -9999em;
+			left: 0;
+			background-color: #fafafa;
+			border-right: 1px solid #eee;
+		}
+
+		li {
+			a {
+				padding: 10px;
+				line-height: 20px !important;
+				border-bottom: 1px solid #eee;
+
+				span {
+					margin-left: 0.618em;
+					margin-right: 0.618em;
+				}
+
+				&::before {
+					@include iconbeforedashicons("\f107");
+				}
+			}
+		}
+
+		&.active a {
+			color: #555;
+			position: relative;
+			background-color: #eee;
+		}
+	}
+}
+
+#woocommerce-coupon-data,
+#woocommerce-product-data {
+	ul.wc-tabs {
+		padding: 0;
+		align-self: stretch;
+
+		li {
+			min-height: var(--grid-unit-50, 40px);
+
+			a {
+				padding: 20px 24px;
+				color: var(--Gray-Gray-60, $studio-gray-60);
+
+				&::before {
+					margin-right: $gap-smallest;
+					vertical-align: middle;
+					display: inline-block;
+					height: 14px;
+				}
+				span {
+					vertical-align: middle;
+				}
+			}
 
 			&.active a {
-				color: #555;
-				position: relative;
-				background-color: #eee;
+				color: var(--Alias-text-text, $alias-text);
+				font-weight: 700;
+				border-left: 3px solid var(--wp-admin-theme-color, $studio-wordpress-blue);
```

Comment:
How about setting the left border to 3px transparent in the following selector? That way, the tab items will be better aligned.

https://github.com/woocommerce/woocommerce/blob/daf105933d5bb378bcecb75afff70bd36ef51c33/plugins/woocommerce/client/legacy/css/admin.scss#L6354

![1](https://github.com/user-attachments/assets/e8b204c0-a04f-4ab8-b04b-a4783a1d2bef)

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/52632/commits/8b022a28d2a94323c78e436aa566793e39bdf668